torch/fx/experimental/symbolic_shapes.py

```diff
@@ -410,9 +489,7 @@ def create_symbolic_sizes_strides(self, ex: torch.Tensor):
         return [self.create_symintnode(i) for i in size], [self.create_symintnode(i) for i in stride]  # type: ignore[arg-type]
 
     def create_symintnode(self, expr: Union["sympy.Expr", int]):
-        py_sym_int = PySymInt(expr, self)
-        cpp_sym_int = torch.SymIntNode.new_symint(py_sym_int)  # type: ignore[attr-defined]
-        return cpp_sym_int
+        return SymInt(SymNode(expr, self, int))
```

Comment:
Do we prefer `wrap_node(SymNode(expr, self, int))` ?

Reply:
it doesn't really matter